<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraToolsClass</class>
 <widget class="QMainWindow" name="CameraToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头信息</string>
    </property>
    <widget class="QComboBox" name="comboBox_Camera">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分辨率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>图像位深：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帧率：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DataType">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DataResolution">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DataBit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DataFps">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>检测工具：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DetectTools">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_StartShow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_picture">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>显示图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Grab">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>抓拍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addmask">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加水印</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_transparency">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartCapture">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录像</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_record">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作记录：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ImageStyle">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图像风格：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Filter">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>噪点优化</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>检测工具</string>
    </property>
    <addaction name="actionDirectShow"/>
    <addaction name="actionMedia_Foundation"/>
    <addaction name="actionVFW"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDirectShow">
   <property name="text">
    <string>DirectShow</string>
   </property>
  </action>
  <action name="actionMedia_Foundation">
   <property name="text">
    <string>Media Foundation</string>
   </property>
  </action>
  <action name="actionVFW">
   <property name="text">
    <string>VFW</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CameraTools.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_StartShow</sender>
   <signal>clicked()</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>button_startshow_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Grab</sender>
   <signal>clicked()</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>button_grab_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Camera</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>combobox_camera_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_addmask</sender>
   <signal>clicked()</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>button_addmask_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_transparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>slider_value_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_StartCapture</sender>
   <signal>clicked()</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>button_startcapture_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_ImageStyle</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>combobox_imagestyle_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1002</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_DetectTools</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>combobox_detecttools_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Filter</sender>
   <signal>clicked()</signal>
   <receiver>CameraToolsClass</receiver>
   <slot>checkbox_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>1052</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_startshow_click()</slot>
  <slot>button_grab_click()</slot>
  <slot>combobox_camera_change()</slot>
  <slot>button_addmask_click()</slot>
  <slot>slider_value_change()</slot>
  <slot>button_startcapture_click()</slot>
  <slot>combobox_imagestyle_change()</slot>
  <slot>combobox_detecttools_change()</slot>
  <slot>checkbox_filter()</slot>
 </slots>
</ui>
